PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -2953,6 +2953,31 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 			return _pluginsManager.allocateIndicator(static_cast<int32_t>(wParam), reinterpret_cast<int *>(lParam));
 		}
 
+		case NPPM_GETTABCOLORID:
+		{
+			const auto view = static_cast<INT>(wParam);
+			auto tabIndex = static_cast<INT>(lParam);
+			auto colorId = -1;  // no color (or unknown)
+			auto pDt = _pDocTab;  // active view
+			if (view == MAIN_VIEW)
+			{
+				pDt = &_mainDocTab;
+			}
+			else if (view == SUB_VIEW)
+			{
+				pDt = &_subDocTab;
+			}

```

Comment:
Please add an empty line for the sake of readability.

Reply:
I added more empty lines throughout; hopefully it is more appealing.